From: Kefeng Wang <wangkefeng.wang@huawei.com>
Subject: page_idle: kill page idle and young wrapper
Date: Fri, 10 Nov 2023 11:33:24 +0800

Since all the calls of page idle and young functions are gone, let's
remove all the wrappers.

Link: https://lkml.kernel.org/r/20231110033324.2455523-8-wangkefeng.wang@huawei.com
Signed-off-by: Kefeng Wang <wangkefeng.wang@huawei.com>
Cc: David Hildenbrand <david@redhat.com>
Cc: Gregory Price <gregory.price@memverge.com>
Cc: Matthew Wilcox (Oracle) <willy@infradead.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/page_idle.h |   25 -------------------------
 1 file changed, 25 deletions(-)

--- a/include/linux/page_idle.h~page_idle-kill-page-idle-and-young-wrapper
+++ a/include/linux/page_idle.h
@@ -119,29 +119,4 @@ static inline void folio_clear_idle(stru
 }
 
 #endif /* CONFIG_PAGE_IDLE_FLAG */
-
-static inline bool page_is_young(struct page *page)
-{
-	return folio_test_young(page_folio(page));
-}
-
-static inline void set_page_young(struct page *page)
-{
-	folio_set_young(page_folio(page));
-}
-
-static inline bool test_and_clear_page_young(struct page *page)
-{
-	return folio_test_clear_young(page_folio(page));
-}
-
-static inline bool page_is_idle(struct page *page)
-{
-	return folio_test_idle(page_folio(page));
-}
-
-static inline void set_page_idle(struct page *page)
-{
-	folio_set_idle(page_folio(page));
-}
 #endif /* _LINUX_MM_PAGE_IDLE_H */
_
